constrain/library/vav_turndown_during_reheat.py

```diff
@@ -44,13 +46,11 @@ class VAVTurndownDuringReheat(RuleCheckBase):
     ]
 
     def verify(self):
-        # Make sure every value in `V_dot_VAV_max` is greater than 0
-        assert (
-            self.df["V_dot_VAV_max"] > 0
-        ).all(), "Not all `V_dot_VAV_max` values are greater than 0"
-
-        # Check if the `reheat_coil_flag` column has only False values
-        if (self.df["reheat_coil_flag"] == False).all():
+        # Check if the `reheat_coil_flag` column has only True/False value
+        if self.df["reheat_coil_flag"].nunique() == 1:
+            self.df["result"] = "Untested"
+        elif (self.df["V_dot_VAV_max"] == 0).any():
+            logging.error("Any `V_dot_VAV_max` value shouldn't be zero.")
```

Comment:
@leijerry888 Based on our discussion, I used logging to catch the 0 division and added ```Untested``` results in the result dataframe.

Reply:
Based on discussion, revert these changes